assets/UI/ui_panels/pixel_1_all/pixel_ui_theme/PixelTheme.tres

```diff
@@ -117,7 +117,7 @@ bg_color = Color(0.556863, 0.556863, 0.556863, 1)
 draw_center = false
 anti_aliasing = false
 
-[sub_resource type="Image" id="Image_kkv03"]
+[sub_resource type="Image" id="Image_5t74u"]
```

Comment:
Idk why but can you remove your changes here ?

Reply:
It doesn't change anything usually cause godot reimport it automatically but i will